src/spectrogramplot.h

```diff
@@ -110,3 +115,23 @@ class TileCacheKey
     int zoomLevel;
     size_t sample;
 };
+
+class AnnotationLocation
+{
+public:
+    Annotation annotation;
+
+    AnnotationLocation(Annotation annotation, int x, int y, int width, int height)
+        : annotation(annotation), x(x), y(y), width(width), height(height) {}
+
+    bool isInside(int pos_x, int pos_y) {
+        return (x <= pos_x) && (pos_x <= x + height)
+            && (y <= pos_y) && (pos_y <= y + width);
```

Comment:
```suggestion
        return (x <= pos_x) && (pos_x <= x + width)
            && (y <= pos_y) && (pos_y <= y + height);
```

Reply:
Indeed. Silly me! I'll force-push the fix. Thanks for spotting this.